src/test/scala/uk/gov/hmrc/test/ui/pages/CommonElements.scala

```diff
@@ -283,6 +283,9 @@ trait CommonElements extends BasePage with Matchers {
     "The date you started getting Jobseeker’s Allowance must include a day" -> By.id("value-for-error"),
     "The date you started getting Jobseeker’s Allowance must include a month"   -> By.id("emptyMonth-error"),
     "The date you started getting Jobseeker’s Allowance must include a year" -> By.id("emptyYear-error"),
+    "Yes, claim Jobseeker's Allowance" -> By.id("value"),
+    "Yes, claim Employment and Support Allowance" -> By.id("value"),
+    "continue, missing claim" -> By.id("add-missing-claim-button-id"),
```

Comment:
Can we also capture `no` scenario ?

Reply:
and also for In year it will not have `add-missing-claim-button-id` button, good to capture that scenaio as well ?

Reply:
As discussed, these changes will be covered part of next PR.